recipes/mwclient/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
```

Comment:
Suggested to add a name and version template as the first two lines of the meta.yaml so that the rest of the recipe becomes slightly simpler to maintain
```yaml
{% set name = "mwclient" %}
{% set version = "0.8.4" %}